docs/home/benefits-of-pwa.md

```diff
@@ -0,0 +1,65 @@
+# Benefits of Progressive Web Apps
+
+Building a Progressive Web App has plenty of advantages, from both a developer and user perspective. If you're looking for a summary of what PWAs can bring to the table, the following article covers several benefits, broken down by how they affect either the development experience or the user experience.
+
+If you are unclear on what a Progressive Web App is from a technical perspective, feel free to check out our [Beginner's Guide to Progressive Web Apps.](/home/pwa-intro/)
+
+## Developer
+
+### Multiple Platforms - Single Codebase
+
+Progressive Web Apps are built with web technology, and as a result, run on any platform with minimal changes to a single, primary codebase. If you develop your app with responsive display design in mind, the same code base will work just as well any mobile (Android, iOS) and any native (Windows, OSX) platform.
+
+You can reach as many users as possible while minimizing the amount of work required to maintain your app, easing development flow and saving time in the process.
+
+### Discoverability
+
+Progressive Web Apps can be packaged and delivered to many native app stores, which allows them to be discoverable from two different paths: on the web and through app stores.
+
+By packaging your PWA with PWABuilder and delivering it to as many app stores as possible, your app can find the largest available audience.
+
+### Ease of Updating
+
+Packaged progressive web apps are still served via the web, and make use of the cache to enable offline capability. Because of this, any time the served version of your app is updated, these changes will automatically be reflected in any store listings or installed versions of your app the next time they connect to the network. Re-uploading your app to native app stores is only necessary if you change your Web App Manifest or if you want to change the listing itself.
+
+New content can be delivered quickly and effectively without waiting on native app store policies or processes.
+
+### Variety of Development Options
+
+The only requirement when developing a progressive web app is that you are using a technology that is capable of being deployed to the web. Because the web has been open and iterated on for so long, the number of development options is nearly endless. You can use anything from Vanilla JavaScript, to React, and everything in between.
```

Comment:
> anything from Vanilla JS to React
this doesn't seem like a ton. You can make this more generic and say thousands of frameworks on the front end to any language on the backend, and even use things like web assembly or web gpu for high performance and critical code.